File: source/code/python/authentication/mfa_create_account.py
directory = client.directories.create({
    'name': 'test_dir',
    'description': 'This is a testing directory.'
})
account = directory.accounts.create({
    'username': 'john_smith',
    'given_name': 'John',
    'surname': 'Smith',
    'email': 'john_smith@example.com',
    'password': 'MynameissmithJohnsmith1'
})

File: source/code/nodejs/authentication/mfa_create_account.js
var accountData = {
  givenName: 'Joe',
  surname: 'Stormtrooper',
  username: 'tk421',
  email: 'tk421@stormpath.com',
  password: 'Changeme1'
};

var application; // An application, fetched from Client.getApplication();

application.createAccount(accountData, function (err, account) {
  if (err) {
    return console.error(err);
  }

  console.log(account);
});
